web-ui/src/main/resources/ui/app/Visualization.js

```diff
@@ -44,8 +44,6 @@
  */
  define(function(require) {
```

Comment:
Shiiiieet codacy... Are you never happy? :D 

Shall we lock her up somewhere, @Gerrrr ?